components/core/Dockerfile.prod

```diff
@@ -19,7 +19,7 @@ RUN apt-get update && \
     apt-get clean
 
 # Installing the python modules
-RUN python3 setup.py develop
+RUN ["/bin/bash", "-c", "python3", "setup.py", "develop"]
```

Comment:
Just curious, why do we need this change? Isn't docker doing the same with Shell form?

Reply:
@agentmilindu 
True this is weird but I have to say this,

I have my docker run failing on my Mac if I do it with bash 
But it is other way round on Travis where it is failing due to `sh`.
What I feel is it's better to push it manually with sh itself. 
I am not sure why this is happening, do you think of any other workaround.

Also, when I approached a few dockerfile maintainers, and what I learned was they do the docker push part manually always.  
Please let me know your views?

Reply:
Add a comment and we can keep